src/BookReader.js

```diff
@@ -1279,7 +1279,8 @@ BookReader.prototype.updateFirstIndex = function(
   if (this.options.initialSearchTerm && !suppressFragmentChange) {
     this.suppressFragmentChange = false;
   }
-  this.trigger('pageChanged');
+
+  this.trigger(BookReader.eventNames.pageChanged, { hasPageChanged: true });
```

Comment:
Q: when page changes - `hasPagedChanged` is always true. feels like we do not need that custom value.

Reply:
`BookReader.eventNames.pageChanged` actually being called from multiple places and even execute when you reload the page as well. so I need to catch specific event when user actually Click on any book page or change the page using br controls. so I found this function is appropriate from the actual fragementChange event does when user interact.

Reply:
resolving to re-think abstraction